README.md

```diff
@@ -6,18 +6,29 @@ This repository contains an implementation example on how to deploy the Quorum K
 
 ## 1.1. Quickstart
 
-1. To deploy a simple Quorum Key Manager (not production ready), run the following command:
+1. To deploy Quorum Key Manager solution, do the following:
+
 
 ```bash
+# Adapt the manifests.yaml with your own needs and values
+# then build up the B64_MANIFESTS like so
+export B64_MANIFESTS=$(envsubst < manifests/manifests.yaml | base64)
```

Comment:
Out of curiosity, why you need `envsubst`? isn't it possible to do `$(cat manifests/manifests.yaml | base64)` ?

Reply:
It is used to fill in env values